### PR TITLE
Create how to resize virtual clusters

### PR DESCRIPTION
**Pull request by ritawatson**
This PR provides a how to for resizing virtual clusters.

[Doc Preview](https://deploy-preview-1111--docs-spectrocloud.netlify.app/devx/palette-virtual-clusters/resize-virtual-clusters)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1111#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1111#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:35]: The size you specify cannot be greater than the Cluster Group limits.

Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:35]: I came here to say the same. @tiwarisumit3 I think I have an idea of how to handle this. Adding a suggestion about.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:29]: Rita, for these bullets, let's add another newline in between them so it looks less compressed when displayed.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:19]:
```suggestion
You can resize virtual clusters from the default size of 4 CPU, 4 GiB Memory, 2 GiB Storage. The size you specify cannot be greater than the system-level quota and the tenant user quota. To learn more about resource quotas, refer to the [resource quota](/devx/resource-quota) documentation.
```

The resource quota page has been updated with rich diagrams and examples. That should help address any confusion.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:35]:
```suggestion
4. Click **Cluster Size** and specify new resource allocations for your virtual cluster.
```

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:41]:
```suggestion
To verify your changes, click **Virtual Clusters** in the left **Main Menu** and select the resized cluster. The virtual cluster Overview page displays the new **Allocated Quota** for the cluster.
```

**Review by karl-cardenas-coding — APPROVED** [review]
@ritawatson some minor suggestions that should address @tiwarisumit3 's feedback.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1111#pullrequestreview-1298484698" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1111#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:29]: They each had a new line between them, but I added a break to space them out.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:19]: done.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:35]: deleted second sentence.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:41]: added left.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:35]: fixed.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md]: added suggestion.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md]: added left

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:40]: The size you specify cannot be greater than the system-level quota for system level cluster group like Beehive and the tenant user quota for tenant level cluster groups

Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md]: Added.

File: content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:40]: The size you specify cannot be greater than the system-level quota for a cluster group like Beehive or the user quota for tenant-level cluster groups.

Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md:40]: changed and to or.

Comment (ritawatson) [reply content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md]: changed and to or

**Pull request merged**
